jupyter_scheduler/models.py

```diff
@@ -289,3 +306,9 @@ class JobFeature(str, Enum):
     output_filename_template = "output_filename_template"
     stop_job = "stop_job"
     delete_job = "delete_job"
+
+
+class Event(BaseModel):
+    event_id: str
+    event_type: str
+    parameters: Dict[str, str]
```

Comment:
Should we have the type here as `Dict[str, Any]`?

Reply:
Yes, good catch, thank you